ConnectorNavisworks/ConnectorNavisworks/Bindings/ConnectorNavisworksBindings.Filters.cs

```diff
@@ -68,20 +66,31 @@ public override List<ISelectionFilter> GetSelectionFilters()
       filters.Add(savedViewsFilter);
     }
 
-    var clashPlugin = _doc.GetClash();
-    var clashTests = clashPlugin.TestsData;
-
-    var groupedClashResults = clashTests?.Tests.Select(GetClashTestResults).Where(x => x != null).ToList();
+    try
+    {
+      var clashPlugin = _doc.GetClash();
+      var clashTests = clashPlugin?.TestsData;
 
-    if (groupedClashResults?.Count >= 0)
+      if (clashTests != null)
+      {
+        var groupedClashResults = clashTests.Tests.Select(GetClashTestResults).Where(x => x != null).ToList();
+
+        if (groupedClashResults.Count >= 0)
+        {
+          //  var clashReportFilter = new TreeSelectionFilter
```

Comment:
commented out code :pained_face

Are you wanting to retain this code for when we DO want to support this? If so, probably better to create or add to an issue and put the code there... I do wonder what the code is doing if in the inner most if block we don't actually do anything :D

Reply:
This is a not-yet-supported feature. It was in earlier releases but with suboptimal results.